src/commands/install.yml

```diff
@@ -6,13 +6,21 @@ parameters:
     type: "string"
     description: "Specify version of terraform to install."
     default: "0.13.5"
+  os:
+    type: string
+    description: "specify the operating system version to install [linux, darwin, windows]"
+    default: "linux"
+  arch:
+    type: string
```

Comment:
Same as above.

Reply:
@KyleTryon I like this approach thx for calling out  👍🏽 I'll make the change

Reply:
@KyleTryon I adjusted the params based on your suggestion. thx